ui-validation/src/main/kotlin/edu/stanford/spezi/ui/validation/CompositionLocal.kt

```diff
@@ -0,0 +1,9 @@
+package edu.stanford.spezi.ui.validation
+
+import androidx.compose.runtime.compositionLocalOf
+
+val LocalValidationEngine =
```

Comment:
I see we provide this internally, is this expected to be configured from the callers? If so, we should not overwrite it would our own, if not we can mark it internal.

What about LocalValidationEngineConfiguration? I see no provide for it

Reply:
We need to expose this as it allows to write custom components similar to ValidatedTextField